typescript/ws-protocol-examples/src/examples/mcap-record.ts

```diff
@@ -129,6 +129,7 @@ async function main(
         SubscriptionId,
         { messageCount: number; mcapChannelId: McapChannelId }
       >();
+      const knownChannelIds = new Set<WsChannelId>();
```

Comment:
`activeChannelIds` would feel more appropriate, since we're clearing on `unadvertise`

Reply:
```suggestion
      const activeChannelIds = new Set<WsChannelId>();
```